diff --git ./src/Obelisk/Asset/Accept.hs ./src/Obelisk/Asset/Accept.hs
index 96ba79b5..9201a817 100644
--- ./src/Obelisk/Asset/Accept.hs
+++ ./src/Obelisk/Asset/Accept.hs
@@ -41,7 +41,6 @@ import Data.List (sort)
 import Data.Map (Map)
 import qualified Data.Map as Map
 import Data.Maybe (catMaybes, fromMaybe, listToMaybe)
-import Data.Monoid ((<>))
 import Data.Ord (Down(..))
 import Data.Proxy (Proxy(..))
 import Data.String (IsString)
diff --git ./src/Obelisk/Asset/Serve/Snap.hs ./src/Obelisk/Asset/Serve/Snap.hs
index 35f2d6d4..548bc381 100644
--- ./src/Obelisk/Asset/Serve/Snap.hs
+++ ./src/Obelisk/Asset/Serve/Snap.hs
@@ -20,12 +20,10 @@ import Control.Applicative ((<|>))
 import Control.Exception (handleJust, try, throwIO)
 import Control.Monad (forM, liftM, unless)
 import Control.Monad.IO.Class (liftIO)
-import Control.Monad.Fail (MonadFail)
 import Data.Attoparsec.ByteString (parseOnly, endOfInput)
 import Data.ByteString (ByteString)
 import qualified Data.ByteString as BS
 import Data.List (isSuffixOf, sort)
-import Data.Monoid ((<>))
 import qualified Data.Text as T
 import Data.Text.Encoding (decodeUtf8, encodeUtf8)
 import System.Directory (getDirectoryContents)
